spendsterIOS/Presentation/Expenses/AddExpenses/AddExpensesViewController.swift

```diff
@@ -0,0 +1,121 @@
+//
+//  AddExpensesViewController.swift
+//  spendsterIOS
+//
+//  Created by Dmytro Holovko on 5/7/19.
+//  Copyright © 2019 KeyToTech. All rights reserved.
+//
+
+import UIKit
+
+class AddExpensesViewController: UIViewController, CategorySelectViewControllelDelegate, AddExpencesView {
+    var choosenCategory: String?
+    var currencyType = "UAH"
```

Comment:
@Thongvor we should get it from presenter

Reply:
no